docs/FAQ.md

```diff
@@ -0,0 +1,106 @@
+
+### CUDA (Default GPU) vs CPU?
+The CPU version of ONNX Runtime provides a complete implementation of all operators in the ONNX spec. This ensures that your ONNX-compliant model can execute successfully (*minor caveat: not 100% of types are supported to minimize the binary size, but can usually be easily added as needed*). On the other hand, not all CUDA kernels are implemented, as these have been prioritized on an as-needed basis. As a result, this means that if your model contains operators that do not have a CUDA implementation, it will fall back to CPU. Switching between CPU and GPU can cause significant performance impact.
+
+If you require a specific operator that is not currently supported, please consider [contributing](./../CONTRIBUTING.md) and/or [file an issue](https://github.com/microsoft/onnxruntime/issues) clearly describing your use case and share your model if possible. 
+
+### TensorRT vs CUDA?
+TensorRT and CUDA are separate execution providers for ONNX Runtime. On the same hardware, TensorRT will generally provide better performance; however, this depends on the specific model and whether the operators in the model can be supported by TensorRT. In cases where TensorRT cannot handle the subgraph(s), it will fall back to CUDA. Note that the TensorRT EP may depend on a different version of CUDA than the CUDA EP. 
+
+### TensorRT/CUDA vs DirectML? 
+DirectML is the hardware-accelerated DirectX 12 library for machine learning on Windows and supports all DirectX 12 capable devices (Nvidia, Intel, AMD). This means that if you are targeting Windows GPUs, using the DirectML Execution Provider is likely your best bet. This can be used with both the ONNX Runtime as well as [WinML APIs](./WinRT_API.md).
+
+## Do the GPU builds support quantized models?
+The default CUDA build does not support any quantized operators right now. The TensorRT EP has limited support for INT8 quantized ops. In general, support of quantized models through ORT is continuing to expand on a model-driven basis. For performance improvements, quantization is not always required, and we suggest trying alternative strategies to [performance tune](./ONNX_Runtime_Perf_Tuning.md) before determining that quantization is necessary.
+
+## Why is the GPU build slower than CPU?
+Depending on which execution provider you're using, it may not have full support for all the operators in your model. Fallback to CPU ops can cause hits in performance speed. Moreover even if an op is implemented by our CUDA execution provider, we may not necessarily assign/place the op to the CUDA EP due to performance reasons. To see the placement decided by ORT, turn on verbose logging and look at the console output.
+
+See:
+* [Issue 1368](https://github.com/microsoft/onnxruntime/issues/1368)
+* [Issue 1246](https://github.com/microsoft/onnxruntime/issues/1246)
+
+## My converted Tensorflow model is so slow - why?
```

Comment:
This doesn't seem like the right question/answer pair. A model could be slow for reasons beyond NCHW.

Reply:
It seems to be a common reason for poor performance from TF models. If there are additional reasons, it would be good to add them as well to explain common things for users to check for. I would like to keep the question as-is, but would be great to expand the answer.